blocks/library/reusable-block/index.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Server-side rendering of the `core/reusable-block` block.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Renders the `core/reusable-block` block on server.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Rendered HTML of the referenced block.
+ */
+function gutenberg_render_block_core_reusable_block( $attributes ) {
+	$reusable_blocks = get_posts( array(
+		'post_type' => 'gb_reusable_block',
+		'name' => $attributes['ref'],
```

Comment:
From https://github.com/WordPress/gutenberg/issues/2081:

> A block would be identified by a UUID which would get saved in post_name. Using a UUID instead of auto-incremented post ID would eliminate the requirement that a block post be created up-front to reserve its ID. The auto-incremented post ID would be for internal use only and not exposed in the API.